packaging/rpm/microshift.spec

```diff
@@ -221,6 +221,26 @@ The microshift-gateway-api-release-info package provides release information fil
 release. These files contain the list of container image references used by Gateway API
 and can be used to embed those images into osbuilder blueprints.
 
+%package observability
+Summary: OpenTelemetry-Collector configured for MicroShift
+BuildArch: noarch
+Requires: microshift = %{version}
+Requires: opentelemetry-collector
+
+%description observability
+Deploys the Red Hat build of Opentelemetry-Collector as a systemd service on host. MicroShift provides client
```

Comment:
Please, fix the case of Opentelemety -> OpenTelemetry to make it consistent with the summary text.